<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music_Songs_Search</class>
 <widget class="QDialog" name="Music_Songs_Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Songs</string>
  </property>
  <widget class="QPushButton" name="SearchSongs_BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchSongs_NumberSearch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SearchSongs_AlbumCommentsValidation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumNumberText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_TitleText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchSongs_AlbumTitleValidation">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="SearchSongs_SongDateBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchSongs_SongTitleBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumTitleText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchSongs_SearchButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchSongs_SongKeyBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumKeyText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Words</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchSongs_TitleSearch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumCommentsText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchSongs_SongNumberBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchSongs_SongCommentsBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_NumberText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchSongs_SaveButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchAlbums_AlbumDateText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Publishing</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchSongs_AlbumKeyValidation">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SearchSongs_BackButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Search</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchSongs_SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Search</receiver>
   <slot>SearchSongs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>882</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SearchSongs_SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>Music_Songs_Search</receiver>
   <slot>SaveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>SearchSongs()</slot>
  <slot>SaveChanges()</slot>
 </slots>
</ui>
